doc/source/building/build_live_iso.rst

```diff
@@ -60,9 +61,9 @@ system from stick do the following:
 
       $ dd if=LimeJeOS-Leap-42.1.x86_64-1.42.1.iso of=/dev/<stickdevice>
 
-3. Run system from stick
+6. Start system from USB stick
 
-   Like in the example above use Qemu to run the system, but attach
+   Like in the example above use QEMU to start the system, but attach
```

Comment:
Ok, if I got you right, you think this topic should be extended with information about booting from a computer system. Shouldn't be a problem to add that.

However, this can only be a bit "vague" as the menu items and the procedure is different on every computer (or BIOS).